src/scrypt.cpp

```diff
@@ -220,6 +228,11 @@ const ::uint32_t
         *hashc = (unsigned char *) &hash,
       //highestZeroBitsSet = 0xe0;
         // Hash target can't be smaller than bnProofOfWorkLimit which is 00000fffff000000
+        // Aren't you saying here that it can't be bigger? Because you're not testing for it!!!
+        // and isn't this directly related to bnProofOfWorkLimit which is a maximum ease, 
+        // or equivalently, a minimum difficulty?
+        // if you actually mean can't be smaller, then this is a maximum difficulty, 
+        // and if the difficulty eases, this can't test for it?
```

Comment:
=> About above comment, you are right, my comment should be: "Hash target can't be bigger than bnProofOfWorkLimit which is 00000fffff000000" (bigger, not smaller)